docs/podman-create.1.md

```diff
@@ -503,6 +503,15 @@ port to a random port on the host within an *ephemeral port range* defined by
 `/proc/sys/net/ipv4/ip_local_port_range`. To find the mapping between the host
 ports and the exposed ports, use `podman port`.
 
+**--pull-always**=*true*|*false*
```

Comment:
Have --pull first, then --pull-always.  Please steal the verbiage from https://github.com/containers/buildah/blob/master/docs/buildah-bud.md.  I thought I'd change in the podman build doc too, but maybe not?